packages/@ionic/cli/src/commands/login.ts

```diff
@@ -157,16 +166,25 @@ If you are having issues logging in, please get in touch with our Support[^suppo
       this.env.config.set('tokens.telemetry', generateUUID());
     }
 
-    if (sso) {
+    if (email && password) {
+      await this.env.session.login(email, password);
+    } else if (sso) {
       this.env.log.info(
-        `Ionic SSO Login\n` +
-        `During this process, a browser window will open to authenticate you with the identity provider for ${input(email)}. Please leave this process running until authentication is complete.`
+        `Ionic SSO Login (DEPRECATED)\n` +
+        `Please run ${input('ionic login')} instead.\n` +
+        `During this process, a browser window will open to authenticate you with the identity provider. Please leave this process running until authentication is complete.`
       );
       this.env.log.nl();
 
       await this.env.session.ssoLogin(email);
     } else {
-      await this.env.session.login(email, password);
+      this.env.log.info(
+        `Ionic Login\n` +
+        `During this process, a browser window will open to authenticate you. Please leave this process running until authentication is complete.`
+      );
+      this.env.log.nl();
```

Comment:
The user no longer gets to read this message before we send them to the browser. The old `ssoLogin` used to prompt for an email which would allow the user to see the message.

Reply:
Maybe we can do a prompt similar to https://github.com/ionic-team/ionic-cli/blob/develop/packages/@ionic/cli/src/commands/link.ts#L382-L393